R/properties.R

```diff
@@ -138,22 +138,47 @@ addQuotesOrComma <- function(v, term)
 }
 #' Copy attributes from one object to another
 #'
-#' Copies the "label", "name", "question" and "questiontype" attributes
-#' for each for variable in a \code{\link{data.frame}}.
+#' Copies attributes such as "label", "name", "question" and "questiontype"
+#' from one object to another. If both objects are data frames,
```

Comment:
How about "If both objects are lists [data frames], elements [columns] in the recipient list [data frame] will also receive attributes from elements [columns] with the same name in the donor list [data frame]."